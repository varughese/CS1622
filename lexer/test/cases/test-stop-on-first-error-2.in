this is 9a error
