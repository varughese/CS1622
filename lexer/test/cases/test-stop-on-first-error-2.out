(1,ID,"this")
(1,ID,"is")
(1,NUM,"9")
(1,ID,"a")
(1,ID,"error")
